docs/learn-staking.md

```diff
@@ -32,23 +30,20 @@ validator set.
 
 In staking, you can be either a [nominator or a validator](#validators-and-nominators).
 
-As a nominator, you can nominate one or more (up to 16) validator candidates that you trust to help
-you earn rewards in DOT. You can take a look at the [nominator guide](learn-nominator) to understand
-what you are required to do when the mainnet launches.
-
-A validator node is required to be responsive 24/7, perform its expected duties in a timely manner,
-and avoid any slashable behavior. You can follow our
-[Kusama validator guide](https://guide.kusama.network/docs/en/mirror-maintain-guides-how-to-validate-kusama)
-if you would like to run a validator on Kusama.
+As a nominator, you can nominate validator candidates that you trust to help you earn rewards in the
+chain's native token. You can take a look at the [nominator guide](learn-nominator) to understand
+your responsibilities as a nominator, and the [validator docs](learn-validator) to understand what
+you need to do as a validator.
 
 ### 2. Nomination period
 
 Any potential validators can indicate their intention to be a validator candidate. Their candidacies
 are made public to all nominators, and a nominator in turn submits a list of any number of
-candidates that it supports. In the next epoch (lasting several hours), a certain number of
-validators having the most DOT backing get elected and become active. There are no particular
-requirements for a DOT holder to become a nominator, though we expect each nominator to carefully
-track the performance and reputation of validators.
+candidates that it supports. In the next epoch (lasting 4 hours on Polkadot, 1 hour on Kusama), a
```

Comment:
nitpick: In the next epoch (each epoch lasts 4 hours on Polkadot, 1 hour on Kusama),

Reply:
Nyeh :D